scripts/MediaSessionManager.js

```diff
@@ -0,0 +1,28 @@
+// Handles MediaSession (audio metadata) management
+
+'use strict';
+
+function MediaSessionManager() {
+	
+	let gameNames = {
+		'animal-crossing': 'Animal Crossing',
+		'wild-world': 'Animal Crossing: Wild World',
+		'new-leaf': 'Animal Crossing: New Leaf',
+	}
+	let gameArtwork = {
+		'animal-crossing': 'https://i.imgur.com/dgxbXpm.png',
+		'wild-world': 'https://i.imgur.com/U9MW9J7.png',
+		'new-leaf': 'https://i.imgur.com/nfpxWtV.png'
+	}
```

Comment:
What do you guys think about these covers? I aimed to leave out all labels from the original covers while still featuring the Animal Crossing logos.

Reply:
They look good! BTW, we probably shouldn't read them from Imgur, we should read them locally.

Reply:
I originally tried setting a relative path to the image, but it didn't work. Chrome complained:
```
MediaImage src can only be of http/https/data/blob scheme
```
As can be seen [here](https://code.immersaview.com/remotes/chromium-third_party/blob/d2bb3bb52ed2e529511b71b85331181ec7fb5464/WebKit/Source/modules/mediasession/MediaMetadataSanitizer.cpp) it looks like Chrome implementation of MediaSessionAPI is expecting a URL in that field. 

More examples here:
https://googlechrome.github.io/samples/media-session/audio.html

Reply:
Well, that's kind of dumb. We can use the data scheme and encode the albums in Base64. I think we should try and keep them offline personally.

Reply:
Great suggestion. I just tried it and it loads base64 artwork flawlessly. Will push changes soon! 👍

Reply:
Already way ahead of you. I'll push everything when I'm finished.